src/bindings/c/tests/ov_core_test.cpp

```diff
@@ -203,7 +205,17 @@ TEST_P(ov_core, ov_core_set_property_invalid_number_property_arguments) {
 
     ov_core_free(core);
 }
-
+/*
+* Skipping the following tests for ARM CPU plugin:
+* 'ov_core_set_property_enum_invalid', cause the property doesn't support by ARM plugin
+* 'ov_core_set_and_get_property_enum', cause the property doesn't support by ARM plugin
+* 'ov_core_set_and_get_property_bool_invalid', cause the property doesn't support by ARM plugin
+* 'ov_core_set_get_property_str', cause the property doesn't support by ARM plugin
+* 'ov_core_set_multiple_common_properties', cause the property doesn't support by ARM plugin
+* 'ov_compiled_model_export_model', cause this feature doesn't support by ARM plugin
+* 'ov_core_import_model', cause this feature doesn't support by ARM plugin
+*/
```

Comment:
- `ov_core_set_property_enum_invalid` - can we use `ov::hint::performance_mode` ? It's supported by ARM
- `ov_core_set_and_get_property_bool_invalid` - here we use enable profiling, it's supported by ARM.
maybe other tests with properties can also use common properties supported by ARM? The list of ARM supported properties is here https://github.com/openvinotoolkit/openvino_contrib/blob/master/modules/arm_plugin/src/arm_plugin.cpp#L151-L203

For Import / export tests like `ov_compiled_model_export_model` - can we check in test that optimization capabilities property does not contain import_export flag and skip tests in runtime? Here is how it's defined for ARM https://github.com/openvinotoolkit/openvino_contrib/blob/master/modules/arm_plugin/src/arm_plugin.cpp#L194-L200

Reply:
For 'ov_core_set_property_enum_invalid' & 'ov_core_set_and_get_property_bool_invalid', ARM plugin doesn't check the value, so for invalid value the interface also return true. (caused the test failed) I think can we add this checking in ARM plugin.
For 'ov_core_set_and_get_property_enum', ARM plugin only have one enum property ov::hint::performance_mode, but in the test case has two enum properties, can we remove one?
For 'ov_core_set_get_property_str', ARM plugin do not have string property, I think we need to skip this.
For 'ov_core_set_multiple_common_properties', which includes the string property caused the test failed.
For 'ov_compiled_model_export_model' & 'ov_core_import_model', will try your proposal.
How do u think @ilya-lavrenov

Reply:
> For 'ov_core_set_property_enum_invalid' & 'ov_core_set_and_get_property_bool_invalid', ARM plugin doesn't check the value, so for invalid value the interface also return true. (caused the test failed) I think can we add this checking in ARM plugin.

Can we send a PR to ARM plugin to have correct checks?

> How do u think @ilya-lavrenov

Ok with the rest

Reply:
get it! Will create an another PR for ARM plugin.

Reply:
@ilya-lavrenov  According our alignments, had finished the change, pls help review.
**One thing need take care, this PR works need the PR (https://github.com/openvinotoolkit/openvino_contrib/pull/582) merged.**

Reply:
thanks!